tests/assertions.js

```diff
@@ -21,22 +20,17 @@ export const testCredential = ({credential}) => {
   credential.should.be.an('object');
   credential.should.have.property('@context');
   credential['@context'].should.include.members([
-    'https://www.w3.org/2018/credentials/v1',
-    'https://w3id.org/vc/status-list/2021/v1',
-    'https://w3id.org/security/data-integrity/v2'
+    'https://www.w3.org/ns/credentials/v2',
+    // 'https://w3id.org/security/data-integrity/v2'
```

Comment:
Are you planning on bringing this back soon? If not, just remove it as it's unclear why it's commented out.

Reply:
This assertion is meant to test the vcdm base features, I will actually take it out.

The statement this address is:

> Any expression of the data model in this section MUST be expressed in a conforming verifiable credential as defined in [VC-DATA-MODEL-2.0].

I don't think checking for `https://w3id.org/security/data-integrity/v2` is part of this conformance requirement.